instances/atlas/URL/JBA-v1-18/JBA-v1-18_Colin27-1998_Area-1_PM-v8-4_left-jsonld.jsonld

```diff
@@ -0,0 +1 @@
+{"@context": {"@vocab": "https://openminds.ebrains.eu/vocab/"}, "@id": "https://openminds.ebrains.eu/instances/URL/JBA-v1.18_Colin27-1998_Area-1_PM-v8.4_left", "@type": "https://openminds.ebrains.eu/core/URL", "URL": "https://atlases.ebrains.eu/viewer/#/a:juelich:iav:atlas:v1.0.0:1/t:minds:core:referencespace:v1.0.0:7f39f7be-445b-47c0-9791-e971c0b6d992/p:minds:core:parcellationatlas:v1.0.0:94c1125b-b87e-45e4-901c-00daee7f2579/r:jubrain%20colin%20v18%20left::1z/@:0.0.0.-W000.._eCwg.2-FUe3._-s_W.2_evlu..7LIx..2pi7c~.1hyh1~.3SUsT..1LSm"}
```

Comment:
should we maybe have indented jsonld here (and other service link instances?)

Reply:
yes... I don't know why it wasn't indented... I will fix that. 
What do you mean with "and other service link instances"?